commands/fly.go

```diff
@@ -26,7 +26,7 @@ type FlyCommand struct {
 	Watch   WatchCommand   `command:"watch"   alias:"w" description:"Stream a build's output"`
 
 	Containers ContainersCommand `command:"containers" alias:"cs" description:"Print the active containers"`
-	Hijack     HijackCommand     `command:"hijack"     alias:"intercept" alias:"i" description:"Execute a command in a container"`
+	Intercept  InterceptCommand  `command:"intercept"     alias:"intercept" alias:"i" description:"Execute a command in a container"`
```

Comment:
I can...it kinda defeats the purpose of the PR, in my mind. Would it be better to sit on the PR and wait for a new version to be released so it can come as a new release?